assets/scss/base/_helpers.scss

```diff
@@ -224,3 +225,8 @@
   }
 
 }
+
+.truncate--fixed-small {
```

Comment:
This name is confusing - why does it says "fixed-small" where the style actually makes the width flexible with max threshold?
Based on styles above I would rather call it `.max-width--small`